Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -136,6 +136,13 @@ The specification can be applied to any software to measure and reduce its carbo
 
 The target audience for this are technical stakeholders (e.g. software architects, developers, and maintainers) who ideally can use this as a methodology so that they can understand the characteristics of their software solution and minimize the associated emissions.
 
+## Software Sustainability Actions
```

Comment:
This looks like the same changes as in https://github.com/Green-Software-Foundation/software_carbon_intensity/pull/43 ? Is this redundant?